views/pages/login.handlebars

```diff
@@ -0,0 +1,9 @@
+<form method="post" action="/login">
+    <div class="field">
```

Comment:
Misschien kan je hier labels van maken ipv divs, dat is wat semantischer

Reply:
De fields zijn voornamelijk bedoelt om related info te groeperen en voor styling purposes. Want ik heb ook al fields met daarin 3 labels en 3 inputs, dus dat zou betekenen dat ik in een 1 label 3 labels heb staan. Ik zou we kijken naar een andere element wat meer van toepassing zou komen.